### ISSUE TITLE
Allow setting Vaadin license key via plugin preferences

### ISSUE THREAD

**Issue by johndevs** [johndevs/gradle-vaadin-plugin#178]
Vaadin license keys are normally stored in a file in the users home directory. This is a bit problematic as when using a CI server you might not want to put the key on the server but instead pass it through the build properties.

**Comment by whiteancient** [comment]
Hello John,

I am trying to use the Spreadsheet-addon. I have the licence in my home directory, but the widgetset is still not compiling. Is there something more I need to do?

**Comment by johndevs** [comment]
Having the license in the home directory should work, have you named the license file correctly? (should be **.vaadin.spreadsheet.developer.license**). 

Vaadin Spreadsheet also requires a Vaadin 7.4.+ so ensure you also are using that.

Here is my test build.gradle if it helps:

```java
apply from: 'http://plugins.jasoft.fi/vaadin.plugin'

vaadin {
    version '7.4.+'
    widgetset 'com.example.MyWidgetset'
}

dependencies {
    compile 'com.vaadin.addon:vaadin-spreadsheet:1.1.0.alpha2'
}
```

**Comment by whiteancient** [comment]
I named the licence-file wrong (there is no dot in the start on the licence-file when I downloaded it from vaadin.com). But that didnt solve it. I use 7.5.0 and have 10+ other plugins that works great. But if it works nice for you its all in my builds. I will look into this further in a few weeks time when the spreadsheet functionality will be essential. I use intellij:s gradle-plugin if that makes any difference, even if it shouldnt...
Happy friday! :)
